### ISSUE TITLE
Choose file - 'more details' link needs to be accessible

### ISSUE THREAD

**Issue by timstone** [EnvironmentAgency/data-returns-frontend#88]
Currently there are numerous 'more details' links when multiple files are uploaded and validation errors have been found.

This is not accessible to some users, for example users of screen readers where the 'more details' link could be taken out of context.

Previously services (Waste exemptions have fixed this by using an ID that describes the link'

`<a id="reviewing-change-yourcontactdetails-yourname" class="change-link" href="/dsc/reviews/applicant_contact/5iSLdoSfyQpfvjjYd39LoFSWTpwuATEwhd">Change<span class="visuallyhidden">your name</span></a>`

This is also relevant for the 'remove' link

**Comment by timstone** [comment]
Moved to Jira
